tests/test.sh

```diff
@@ -11,6 +11,12 @@ export ACTION="ci_test_provision"
 
 "${METAL3_DIR}"/tests/run.sh
 
+"${METAL3_DIR}"/tests/scripts/fetch_manifests.sh
+
+export ACTION="pivoting"
+
+"${METAL3_DIR}"/tests/run.sh
+
```

Comment:
Yes the manifest is collected before pivoting. The manifest was collected before pivoting in our feature test but for some reason we were collecting the manifest after repivoting  in case of integration test. Which will fail in case the job is failed during pivotinng or repivoting. To have similar configuration with feature and integration tests